src/V3Os.cpp

```diff
@@ -266,3 +317,12 @@ uint64_t V3Os::memUsageBytes() {
     return (text + data) * getpagesize();
 #endif
 }
+
+void V3Os::usleep(int64_t usec) {
+#if defined(_WIN32) || defined(__MINGW32__)
+    std::this_thread::sleep_for(std::chrono::microseconds(usec));
+#else
+    // cppcheck-suppress obsoleteFunctionsusleep
+    ::usleep(usec);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [This C routine is considered obsolete (as opposed to the shell command by the same name).   The interaction of this function with SIGALRM and other timer functions such as sleep(), alarm(), setitimer(), and nanosleep() is unspecified (CWE-676).  Use nanosleep(2) or setitimer(2) instead. ](https://app.codacy.com/app/wsnyder/verilator/pullRequest?prid=4719042)